### ISSUE TITLE
STIX Two Math support

### ISSUE THREAD

**Issue by kiwi0fruit** [mathjax/MathJax#1969]
New version of STIX font is actually a different font from STIX ver.1. And I think that STIX Two looks nicer than STIX ver.1.

So it would be nice to have it in MathJax v3.

**Comment by dpvc** [comment]
Yes, we have STIX v2 in mind for MathJax v3.  The font infrastructure is being rewritten, and we expect to repackage the fonts as well, though that might not be in the first release of v3.

**Comment by kiwi0fruit** [comment]
I'm curious if there is a progress on this?

**Comment by dpvc** [comment]
The first release of MathJax v3 is nearly ready and we are hoping for a release this month.  It includes SVG and CommonHTML output using the current MathJax TeX fonts.  Work on additional font support is planned for a future release, but the timeline for those has not been worked out yet.

Getting STIX 2 in place will be a big project (the toolchain for getting the proper data for MathJax has to be written, as the data layout for v3 is quite different from that for v2).  But it is expected to be one of the high priorities after the initial release.
